operate/webapp/pom.xml

```diff
@@ -76,6 +76,19 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>camunda-security-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>camunda-security-services</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-gateway-rest</artifactId>
+    </dependency>
```

Comment:
You added this dependency to use `TenantAttributeHolder` in operate-webapp, 
can we rather move it to a shared dependency like `camunda-security-services` ?

Reply:
Why would a security module depend on a rest module ? IMHO rest modules should be top level, nothing depend on them, except other rest modules, like Operate webapp.

Reply:
No. I did not say we should make security module depend on rest module, I said we just need to move `TenantAttributeHolder ` to a shared security module
Maybe to `io.camunda:camunda-authentication` which already used by Operate webapp and also uses Spring